yarn.lock

```diff
@@ -708,6 +708,10 @@ boom@5.x.x:
   dependencies:
     hoek "4.x.x"
 
+bourbon@^4.3.4:
```

Comment:
there should be no lockfile diffs. make sure you pull in master

Reply:
I just did, tried reverting yarn.lock to the master version, ran `yarn`, and these came back?